app/components/person-sok/PersonSok.tsx

```diff
@@ -0,0 +1,79 @@
+import { BodyShort, Search } from "@navikt/ds-react";
+import { useLocation, useRouteLoaderData } from "@remix-run/react";
+import { useEffect, useState } from "react";
+import { type IOppgave } from "~/models/oppgave.server";
+import { type IRootLoader } from "~/root";
+import { RemixLink } from "../RemixLink";
+import styles from "./PersokSok.module.css";
+
+export function PersonSok() {
+  const location = useLocation();
+  const { oppgaver } = useRouteLoaderData("root") as IRootLoader;
+  const [sokResultat, setSokResultat] = useState<IOppgave[]>([]);
+  const [visSokResultat, setVisSokResultat] = useState(false);
+  const [sokInput, setSokInput] = useState("");
```

Comment:
Vi pratet jo om det i sted, men vi burde ta en runde på kodekonvensjoner. Skal vi kjører på med engelsk eller norsk. Forrige gang landet vi vel på engelsk, men med norsk for alt som er domene spesifikt.

Reply:
Per nå har vi kjørt for det meste norsk i dette prosjektet, og det tenker jeg er helt innafor også for denne PRen. Vi kan gjerne se på konvensjonene fremover utenfor denne PRen :)

Reply:
Når vi startet repoet ble vi enige om å bruke engelsk som hovedregel,og norsk for alt som var domene spesifikt. Jeg ser at det har blitt noe mer norsk, men det er jo fortsatt flust med engelsk her.

Jeg tenker vi må ta en prat og bli enige om konvensjonene sånn at det blir lettere å kjøre PR og at vi får en helhetlig kodebase. Det viktige for meg er ikke om det er norsk eller engelsk, bare vi konsekvente er jeg fornøyd.